src/main/java/seedu/address/model/date/Date.java

```diff
@@ -87,7 +87,7 @@ public int compareTo(Date secondEventDate) {
     public boolean isWithinThreeDays() {
         LocalDate taskDue = LocalDate.parse(this.date);
 
-        if (taskDue.isBefore(fourDaysFromNow())) {
```

Comment:
This breaks the predicate to test if a task should be in reminder. If task is overdue, it wouldn't be displayed in the reminder list if following this code.